Projects/TEAM-1/Turbal Y/final/final/Program.cs

```diff
@@ -0,0 +1,221 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Collections;
+using System.IO;
+using System.Xml.Serialization;
+
+/* Створити колекцію транспорту, додати кілька різних видів транспорту та автомобілів до цієї колекції
+реалізуйте виведення автомобілів старіших ніж 10 років
+відсортуйте інформацію за брендом та моделлю
+збережіть колекцію у файл
+реалізуйте опрацювання виключень (Exceptions)
+сереалізуйте (Serialize) колекцію в XML файл
+десереалізуйте колекцію з XML файла
+напишіть unit тести до реалізованих функцій*/
+
+
+namespace final
+{
+    // Создать объект типа IComparer для объектов класса Inventory.
+    class CompInv1 : IComparer
```

Comment:
It's not really good approach to store all your classes in one file. Could you try to split your code to separate files?